spec/views/two_factor_authentication/otp_verification/show.html.slim_spec.rb

```diff
@@ -1,7 +1,15 @@
 require 'rails_helper'
 
 describe 'two_factor_authentication/otp_verification/show.html.slim' do
-  let(:presenter_data) { attributes_for(:generic_otp_presenter) }
+  let(:presenter_data) do
+    {
+      otp_delivery_preference: 'sms',
+      phone_number: '***-***-1212',
+      code_value: '12777',
+      unconfirmed_user: false,
+      reenter_phone_number_path: verify_phone_path,
```

Comment:
That method is defined in the concern (`app/controllers/concerns/two_factor_authenticatable.rb`) -- do not see any specs that exercise the method, as far as I can tell. Where would you expect to see that method tested?